examples/lib/insertPreview.js

```diff
@@ -44,9 +47,17 @@ function updatePreviews(src, selector) {
     }
   };
 
-  Object.keys(previewSequencerSteps).forEach(function (step, index) {
-    generatePreview(step, Object.values(previewSequencerSteps)[index], src, selector);
+  var sequencer = ImageSequencer();
+
+  sequencer.loadImage(src, function(){
+    this.addSteps('resize', {['resize']: '30%'});
```

Comment:
Should we limit to a fixed size? Like 100x100? Because really large images at 30% are still quite large!

Reply:
but idt we have that option in resize.